odltools/monitor/cli.py

```diff
@@ -1,9 +1,45 @@
-def haha(args):
-    print("tim, tim, bo-bim, banana-fana, foo-fim, tiiiimmm")
+# Copyright 2018 Red Hat, Inc. and others. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import odltools.monitor.monitor_odl_cluster as monitor
+
+JSON_FORMAT = """
```

Comment:
could we pull this from the monitor.py file? Seems weird the variable is defined here but commented on in module.py. At the same time, I can see it staying as it is since this is just help text.